client/src/pages/reports/Form.js

```diff
@@ -1423,6 +1457,7 @@ const ReportForm = ({
     const variables = { report }
     return _saveReport(edit, variables, sendEmail).then(response => {
       const report = response[operation]
+      attachmentFunc.current(report.uuid)
```

Comment:
Is `attachmentFunc.current` guaranteed to always be non-null (it is initialised to null)?